src/deepsparse/transformers/pipelines/question_answering.py

```diff
@@ -501,7 +501,20 @@ def route_input_to_bucket(
         :param pipelines: Different buckets to be used
         :return: The correct Pipeline object (or Bucket) to route input to
         """
-        current_seq_len = len(input_schema.question.split())
+        tokenizer = pipelines[0].tokenizer
+        tokens = tokenizer(
+            " ".join((input_schema.context, input_schema.question)),
+            add_special_tokens=True,
+            return_tensors="np",
+            padding=False,
+            truncation=False,
+        )
+
+        current_seq_len = len(tokens)
+        if not TransformersPipeline.buckets_are_sorted_by_sequence_length(
```

Comment:
Why not just sort in all cases? It's a little bit easier to read and more runtime efficient in the needs-to-sort case

Reply:
Alternatively, `find` the sequence. if not found, return numpy.max([pipeline.sequence_length for sequence in pipelines])

Reply:
TLDR There are ways of writing this without writing your own search function or having to sort

Reply:
+1 on calling sort in all cases & deleting buckets_are_sorted function

Reply:
agreed - let's get the route function rewritten to find the pipeline without using sort. will push something up